common/static/common/js/components/views/feedback.js

```diff
@@ -65,6 +67,39 @@
                     return this;
                 },
 
+                inFocus: function() {
+                    this.options.outFocusElm = this.options.outFocusElm || document.activeElement;
+
+                    // Set focus to the container.
+                    $(this.el).find(".wrapper").first().focus();
+
+                    /**
+                     * Make tabs within the prompt loop rather than setting focus
+                     * back to the main content of the page.
+                     */
+                    var tabbables = $(this.el).find(":tabbable");
+                    tabbables.on("keydown", function (event) {
+                        // On tab backward from the first tabbable item in the prompt
+                        if (event.which == 9 && event.shiftKey && event.target === tabbables.first()[0]) {
```

Comment:
If you are using JQuery UI, you can use the key constants:

    if (event.which == $.ui.keyCode.TAB ...